packages/blaze/package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'blaze',
   summary: "Meteor Reactive Templating library",
-  version: '2.6.2',
+  version: '2.7.0-alpha.1',
```

Comment:
Needs to be 2.7

Reply:
in the oficial would be something like:

```
version: '2.7',
```
?